### DISCUSSION TITLE
Do I need to remove the existing OS before reinstalling with Cobbler?

### DISCUSSION THREAD

**Discussion by helloxjade** [cobbler/cobbler#3863] [Q&A] [answered]
I'm trying to use Cobbler for reinstalling a system on a client machine. The client machine already has an existing OS installed. Do I need to remove the current operating system before running the reinstallation, or can Cobbler overwrite it without wiping the existing OS?

**Comment by SchoolGuy** [comment, marked as answer]
This depends on your autoinstallation configuration. Most installers are able to wipe your existing hard drives. Cobbler doesn't wipe your storage per default as this may or may not be desired.
